src/test_workflow/integ_test/integ_test_runners.py

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+
+from test_workflow.integ_test.integ_test_runner_opensearch import IntegTestRunnerOpenSearch
+from test_workflow.integ_test.integ_test_runner_opensearch_dashboards import IntegTestRunnerOpenSearchDashboards
+
+
+class IntegTestRunners:
+    def __klass(name):
+        if name == "OpenSearch Dashboards":
```

Comment:
Let's do even better and future proof this.

```python
RUNNERS = {
  "OpenSearch" : IntegTestRunnerOpenSearch,
  "OpenSearch Dashboards" : IntegTestRunnerOpenSearchDashboards
}
```

Then you can get rid of `__klass` and just `return RUNNERS[name]`. It will fail with a `KeyError`, and if you want to be fancy you can first check whether `name in RUNNERS` and throw a better exception.

We do this here: https://github.com/opensearch-project/opensearch-build/blob/5820881cc214cb6c0a70ba4fd93decbaea324b76/src/manifests/manifest.py#L46